cmd/interchaincmd/messengercmd/deploy.go

```diff
@@ -257,11 +257,11 @@ func CallDeploy(_ []string, flags DeployFlags, network models.Network) error {
 				if err != nil {
 					return err
 				}
-				if teleporterMessengerAddress != "" {
-					clusterConfig.ExtraNetworkData.CChainTeleporterMessengerAddress = teleporterMessengerAddress
+				if messengerAddress != "" {
+					clusterConfig.ExtraNetworkData.CChainTeleporterMessengerAddress = messengerAddress
```

Comment:
`CChainTeleporterMessengerAddress`

Reply:
`CChainTeleporterRegistryAddress`

Reply:
this is persisted data, will keep the same names for backwards compat